### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by ferreirarocha** [manas99/youtube-and-chill#1]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.45.0 x64
**Electron**: 4.2.7
**OS**: "Fedora release
**Thrown From**: [Youtube-and-chill](https://github.com/manas99/youtube-and-chill) package 0.1.2

### Stack Trace

Uncaught TypeError: Cannot read property 'split' of undefined

```
At /home/marcos/.atom/packages/Youtube-and-chill/lib/video-element-view.js:52

TypeError: Cannot read property 'split' of undefined
    at VideoElementView.formatURL (/packages/Youtube-and-chill/lib/video-element-view.js:52:25)
    at Object.showVideo (/packages/Youtube-and-chill/lib/youtube-and-chill.js:53:33)
    at /packages/Youtube-and-chill/lib/youtube-and-chill-view.js:35:25)
```

### Commands

```
     -0:48 markdown-writer:insert-new-line (input.hidden-input)
     -0:47.9.0 editor:newline (input.hidden-input)
     -0:47.8.0 markdown-writer:insert-new-line (input.hidden-input)
     -0:47.8.0 editor:newline (input.hidden-input)
     -0:47.3.0 markdown-writer:insert-new-line (input.hidden-input)
     -0:47.3.0 editor:newline (input.hidden-input)
     -0:46.5.0 core:paste (input.hidden-input)
     -0:45.5.0 editor:select-to-first-character-of-line (input.hidden-input)
     -0:43.9.0 application:open-folder (input.hidden-input)
     -0:41.6.0 youtube-and-chill:enterURL (input.hidden-input)
     -0:38.7.0 core:paste (input.hidden-input)
     -0:18.8.0 core:cut (input.hidden-input)
     -0:16.9.0 core:delete (input.hidden-input)
     -0:16 markdown-writer:insert-new-line (input.hidden-input)
     -0:16 editor:newline (input.hidden-input)
     -0:13.5.0 youtube-and-chill:enterURL (input.hidden-input)
```

### Non-Core Packages

```
markdown-writer 2.11.10 
minimap 4.29.9 
tool-bar 1.2.2 
tool-bar-markdown-writer 0.5.1 
Youtube-and-chill 0.1.2 
```

**Comment by manas99** [comment]
How exactly did you get the error?
The steps to reproduce...

**Comment by larsdevolder** [comment]
I got this error by entering an url and pressing the 'Go' button

**Comment by manas99** [comment]
The problem lies in the format of the URL. It considers that the protocol, 'https' is also entered with the URL. Currently, try prepending the protocol, so that the URL becomes, 'https://youtube.com/watch?v={video_code}' until I fix this!
Thanks!

**Comment by manas99** [comment]
Hey There,
Sorry for the loooong delay...
I have fixed this bug in v0.1.4!
Thanks for bringing this to notice!
